docs/user-guide/spec.md

```diff
@@ -59,8 +59,8 @@ This makes it easy to create a Tamr dataset from a CSV:
 ```python
 import pandas as pd
 
-df = pd.read_csv("my_data.csv")
-dataset = tamr.datasets.create_from_dataframe(df, "primary key name", "My Data")
+df = pd.read_csv("my_data.csv", dtype=str)
```

Comment:
Can you include some prose (either as a comment in the codeblock or outside the codeblock) explaining the need for this? Even one sentence is good!

Reply:
done